pages/workHeader.xml

```diff
@@ -12,450 +12,458 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+               manuscript <att>xml:id</att>. If the manuscript is in already described in Beta
+               maṣāḥǝft, that is enough. If the manuscript is described in an external catalogue,
+               then use <tag>@type='external'</tag>. <egXML xmlns="http://www.tei-c.org/ns/Examples">
+                  <listWit>
+                     <witness type="external" xml:id="P"
+                        corresp="http://digi.ub.uni-heidelberg.de/diglit/cpgraec398/0569"
+                        facs="http://digi.ub.uni-heidelberg.de/diglit/cpgraec398/0569">
+                        <idno>Cod. Pal. graec. 398</idno>
+                        <ptr
+                           target="https://digi.ub.uni-heidelberg.de/diglit/iiif/cpgraec398/manifest.json"
+                        /> Sammelhandschrift — Konstantinopel, letztes Viertel 9. Jh. </witness>
+                     <witness type="external" xml:id="L"
+                        corresp="http://www.bl.uk/manuscripts/Viewer.aspx?ref=add_ms_19391_f009r"
+                        facs="http://www.bl.uk/manuscripts/Viewer.aspx?ref=add_ms_19391_f009r">
+                        <idno>British Library Add MS 19391</idno>
+                     </witness>
+                  </listWit>
+               </egXML> A simpler form, with only the link would also be ok. <egXML
+                  xmlns="http://www.tei-c.org/ns/Examples">
+                  <witness corresp="https://eap.bl.uk/archive-file/EAP286-1-1-133" type="external"/>
+               </egXML>
+            </p>
+            <p> You can use <att>cert</att> to specify a degree of certainty regarding the
+               manuscript you are listing. Please, note that each witness element can take a fully
+               marked-up description about the text.</p>
+            <p>As information provided by the catalogues we are working with should not be lost,
+               whenever a catalogue description contains references to other manuscripts containing
+               the same work, minimal manuscript records containing the work ID should be created
+               for those manuscripts. This assures that all the manuscripts containing the same work
+               can be retrieved by the app while reserving <gi>listWit</gi> in the work records
+               mainly for the manuscripts used for the edition.</p>
+
+            <p>Please note that the list of manuscripts containing a work shown in the application
+               and the list of witnesses used for an edition are two different things. The first
+               picks up whatever information is available, it might be less or more than we know at
+               the moment depending on what has been marked-up and linked. The <gi>listWit</gi>
+               contains instead a statement about the manuscripts used for an edition, which can be
+               nested, and also might contain manuscripts which we do not have yet.</p>
```

Comment:
My impression is that "and also might contain manuscripts which we do not have yet" is now in contradiction to the above described practice of creating stub records. Maybe it could be deleted.